fe1-web/model/objects/Channel.ts

```diff
@@ -11,10 +12,31 @@ export function channelFromIds(...args: Hash[]) : Channel {
       .join('/')}`;
 }
 
+/**
+ * Returns the social channel of the current user.
+ *
+ * @param laoIdHash - The hash containing the laoID of the currently opened LAO
+ */
+export function userSocialChannel(laoIdHash: Hash): Channel {
+  const userPublicKey = KeyPairStore.getPublicKey().valueOf();
```

Comment:
My understanding is that this basically de-anonymizes the user, because `getPublicKey()` is going to be constant over time. Is this desired ? I don't remember the social media specification in this regard.

Reply:
Yes, this is what we chose to do. Each user sends chirp to his own channel, namely root/<laoID>/social/<userPublicKey>. This way, the channel of a user is constant over time.